openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1644,32 +1653,38 @@ The technology described in this specification was made available from contribut
 
-## Credential Offer - Cross-Device (with Information Pre-Submitted by the End-User) {#use-case-4}
+## Credential Offer - Cross-Device (with Information Pre-Submitted by the End-User) {#use-case-2}
 
 The End-User is starting a job at a new employer. An employer has requested the End-User to upload certain documents to the employee portal. A few days later, the End-User receives an email from the employer notifying her that the employee Credential is ready and asking her to scan a QR code to retrieve it. The End-User scans the QR code with her smartphone, which opens her Wallet. Meanwhile, the End-User has received a text message with a PIN code to her smartphone. After entering that PIN code in the Wallet for security reasons, the End-User confirms the Credential issuance, and receives Credential into the Wallet.
 
-## Credential Offer - Cross-Device & Deferred {#use-case-5}
+## Credential Offer - Cross-Device & Deferred {#use-case-3}
 
 The End-User wants to obtain a digital criminal record. She visits the local administration's office and requests the issuance of the official criminal record as a digital Credential. After presenting her ID document, she is asked to scan a QR code with her Wallet. She is being told that the actual issuance of the Credential will take some time due to necessary background checks by the authority. 
 
 In the Wallet, the End-User sees an indication that issuance of the digital record is under way. A few days later, the End-User receives a notification from her Wallet that requested Credential was successfully issued. When the End-User opens the Wallet, she is asked whether she wants to download the Credential. She confirms, and the new Credential is retrieved and stored in the Wallet.
 
-## Wallet Initiated Issuance during Presentation {#use-case-1}
+## Wallet Initiated Issuance during Presentation {#use-case-4}
 
 An End-User comes across a verifier app that is requesting the End-User to present a Credential, e.g., a driving license. The Wallet determines the requested Credential type(s) from the presentation request and notifies the End-User that there is currently no matching Credential in the Wallet. The Wallet selects a Credential Issuer capable of issuing the lacking Credential and, upon End-User consent, sends the End-User to the Credential Issuer's End-User experience (Web site or app). Upon being authenticated and providing consent to issue the Credential into her Wallet, the End-User is sent back to the Wallet. The Wallet informs the End-User that Credential was successfully issued into the Wallet and is ready to be presented to the verifier app that originally requested presentation of that Credential.
```

Comment:
is the `verifier app ` a defined term? It seems not, do we have something to inform the reader about this?